### PR TITLE
recommend items in add-item-form

### PR DESCRIPTION
**Pull request by AnsonLee99**
### Administrative Info
Monday Board ID: 1058461054
Make sure your branch name conforms to: `<feature/staging/hotfix/...>/[username]/[Monday Item ID]-[3-4 word description separated by dashes]`. Otherwise, please rename your branch and create a new PR.

### Changes
I moved all the calls dealing with retieving items from MongoDB from Sidebar.js into Home.js. The items are then passed to the components via props. A list of items and a unique ID is passed into AddButton.js for the recommendation field.

### Testing
Typed in parts of item names and made sure the appropriate suggestions showed up.

### Confirmation of Change 
image

image

### CODE REVIEW

**Review by zainnk902 — APPROVED** [review]
lgtm

File: client/src/home/Home.js [hunk 1]
```diff
@@ -19,10 +23,59 @@ const Home = () => {
...
+      console.log(items_raw);
+      var batches_raw = [];
+      var itemNames = [];
+
+      if (items_raw != null) {
+        items_raw.forEach(item => {
+          // Format item names
+          item.name = item.name.charAt(0).toUpperCase() + item.name.slice(1);
+          itemNames.push({
+            id: itemNames.length,
```
Comment (hahsu) [inline client/src/home/Home.js:56]: I'm just curious, what is this id for, and why did you set it to the current length of the itemNames array?

**Review by hahsu — APPROVED** [review]
I left one question for you to answer just out of curiosity, but otherwise lgtm! I also just wanted to confirm, new items can be added right? For example let's say pineapple is not a currently an item. If the user adds a batch of pineapples, it will show up in the drop down the next time he tries to add pineapples right?

File: client/src/home/Home.js (hunk 1 above)
Comment (AnsonLee99) [reply client/src/home/Home.js:56]: The autocomplete package required a unique ID for each item

**Pull request merged**